sdcm/cluster.py

```diff
@@ -2564,44 +2564,26 @@ def _resharding_status(self, status):
         return bool(found) if status == 'start' else not bool(found)
 
     # Default value of murmur3_partitioner_ignore_msb_bits parameter is 12
-    def _restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int = 12):
+    def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int = 12) -> None:
+        """
+        Resharding is started during Scylla startup and Scylla initialization has to wait until resharding done
+        """
         self.stop_scylla()
         # Change murmur3_partitioner_ignore_msb_bits parameter to cause resharding.
         with self.remote_scylla_yaml() as scylla_yml:
             scylla_yml.murmur3_partitioner_ignore_msb_bits = murmur3_partitioner_ignore_msb_bits
-        search_reshard = self.follow_system_log(patterns=['Reshard', 'Reshap'])
+
+        search_reshard = self.follow_system_log(patterns=[DB_LOG_PATTERN_RESHARDING_START])
         self.start_scylla(timeout=7200)
-        return search_reshard
+        resharding_started = list(search_reshard)
 
-    def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int = 12) -> None:
-        # TODO: reuse 'DB_LOG_PATTERN_RESHARDING_START' and 'DB_LOG_PATTERN_RESHARDING_FINISH' vars here.
-        search_reshard = self._restart_node_with_resharding(murmur3_partitioner_ignore_msb_bits)
-
-        resharding_started = wait.wait_for(func=self._resharding_status, step=5, timeout=180,
```

Comment:
`_resharding_status` function seems to be not used anymore, it can be deleted

Reply:
@fruch it's deleted, you commented on an old code

Reply:
I was referring to this function:
```python
    def _resharding_status(self, status):
        """
        Check is there's Reshard listed in the "nodetool compactionstats" output
        status : expected values: "start" or "finish"
        """
        patt = re.compile('RESHARD|RESHAP')
        result = self.run_nodetool("compactionstats", publish_event=False)
        found = patt.search(result.stdout)
        # wait_for_status=='finish': If 'RESHARD' is not found in the compactionstats output, return True -
        # means resharding was finished
        # wait_for_status=='start: If 'RESHARD' is found in the compactionstats output, return True -
        # means resharding was started
        return bool(found) if status == 'start' else not bool(found)
```

Reply:
removed